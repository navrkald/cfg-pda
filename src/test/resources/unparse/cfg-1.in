A=> 0A1 | e
