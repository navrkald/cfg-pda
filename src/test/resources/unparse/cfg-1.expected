A => 0A1
A => e